generate_golden.sh

```diff
@@ -28,63 +28,85 @@ build_kb() {
+            $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
+            $kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
+            $kb create webhook --group crew --version v1 --kind FirstMate --conversion
+            $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
+        elif [ $project == "project-v2-multigroup" ]; then
+            header_text 'Generating project-v2-multigroup'
+            export GO111MODULE=on
+            export PATH=$PATH:$(go env GOPATH)/bin
+            go mod init sigs.k8s.io/kubebuilder/testdata/project-v2-multigroup  # our repo autodetection will traverse up to the kb module if we don't do this
 
-		$kb init --project-version $version --domain testproject.org --license apache2 --owner "The Kubernetes authors"
-		$kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
-		$kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
-		$kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
-		$kb create webhook --group crew --version v1 --kind FirstMate --conversion
-		$kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
-		# TODO(droot): Adding a second group is a valid test case and kubebuilder is expected to report an error in this case. It
-		# doesn't do that currently so leaving it commented so that we can enable it later.
-		# $kb create api --group ship --version v1beta1 --kind Frigate --example=false --controller=true --resource=true --make=false
-		$kb create api --group core --version v1 --kind Namespace --example=false --controller=true --resource=false --namespaced=false --make=false
-		# $kb create api --group policy --version v1beta1 --kind HealthCheckPolicy --example=false --controller=true --resource=true --namespaced=false --make=false
-	fi
-	make all test # v2 doesn't test by default
-	rm -f Gopkg.lock
-	rm -f go.sum
-	rm -rf ./vendor
-	rm -rf ./bin
-	export GOPATH=$oldgopath
-	popd
+            header_text 'initializing  project-v2-multigroup ...'
+            $kb init --project-version $version --domain testproject.org --license apache2 --owner "The Kubernetes authors"
+
+            header_text 'Switching to multigroup layout ...'
+            $kb edit --multigroup
+
+            header_text 'Creating APIs ...'
+            $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
+            $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
+            $kb create api --group ship --version v1 --kind FirstMate --controller=true --resource=true --make=false
```

Comment:
I'd suggest exercise besides using `v1` as version. And also I'd suggest tweaking the naming, though it's not critical. e.g. <group: ship, kind: FirstMate> seems to be strange. [These](https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/gopath/src/project/pkg/apis) are the groups and kinds we are using in v1 testdata.
Probably you can do something like this:
```
            $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
            $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
            $kb create api --group crew --version v1beta1 --kind FirstMate --controller=true --resource=true --make=false
            $kb create api --group ship --version v1 --kind Frigate --controller=true --resource=true --make=false
            $kb create webhook --group ship --version v1 --kind Frigate --conversion
            $kb create api --group ship --version v2alpha1 --kind Cruiser --controller=true --resource=true --namespaced=false --make=false
            $kb create api --group example-app --version v1 --kind Foo --controller=true --resource=true --make=false
```

Reply:
Done. Please, let me know if you liked it now.

Reply:
> <group: ship, kind: FirstMate> seems to be strange.

I mean that `FirstMate` is not a kind of ship. This part is strange.
You new changes <ship, Sailor> and <ship, Admiral> IMO still have this problem.

Sorry about bikeshedding this :P

Reply:
Oh .. catcher that . This are kind of ships and not roles on the ship :-) Fixing it now.